wpilibc/src/main/native/cpp/geometry/Rotation2d.cpp

```diff
@@ -23,11 +25,7 @@ Rotation2d::Rotation2d(double x, double y) {
     m_sin = 0.0;
     m_cos = 1.0;
   }
-  m_value = std::atan2(m_sin, m_cos);
-}
-
-Rotation2d Rotation2d::FromDegrees(double degrees) {
-  return Rotation2d(Deg2Rad(degrees));
+  m_value = units::radian_t(std::atan2(m_sin, m_cos));
```

Comment:
units::math::atan2() should work for this.

Reply:
It complains that the dimensionality of y/x does not reduce to dimensionless when using doubles. It makes most sense to take in doubles here because it's a ratio and can represent lengths, velocities, etc.